target/linux/ramips/dts/YOUKU-YK2.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "youku,yk2", "mediatek,mt7621-soc";
+	model = "YOUKU YK2";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "youku-yk2:blue:power";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
```

Comment:
You can add a default-state = "on" line here to set default led state.
BTW all leds are off by default so you don't need to add default-state="off" for wps led.

Reply:
ok , will add it.